nvi-commons/src/main/java/no/sikt/nva/nvi/common/model/NviCandidateIndexDocument.java

```diff
@@ -0,0 +1,92 @@
+package no.sikt.nva.nvi.common.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.util.List;
+import java.util.Objects;
+import nva.commons.core.JacocoGenerated;
+
+public final class NviCandidateIndexDocument {
```

Comment:
If the fields have getters, this is a record. Again, why the same class as in `no.sikt.nva.nvi.index.model`?

Reply:
I think to implement search in other repo than indexing, but it was not the best idea, so i removed everything from commons now.